services/ui-src/src/components/pages/Export/ExportedReportPage.test.tsx

```diff
@@ -19,7 +19,7 @@ describe("reportTitle", () => {
   it("should generate the correct title for WP report type", () => {
     const reportType = ReportType.WP;
     const reportPage = { heading: "Report Heading" };
-    const report: ReportShape = {
+    const report: any = {
```

Comment:
Why is the report type getting changed to `any`?

Reply:
Somehow reverted to an old version I had. It should be good now.